README.md

```diff
@@ -11,6 +11,7 @@
 * [8. Setup Remote Attestation on Host OS and inside TD](#setup-remote-attestation)
 * [9. Build Packages From Source](#build-packages-from-source)
 * [10. Run Tests](#sanity-functional-tests)
+* [11. Secure boot](#secure-boot)
```

Comment:
1) Change title to "Secure Boot TD"
2) I think this can be a subsection of "6. Boot TD" since it's booting TD related.  So it would be 6.3.  If you think it deserves a section of its own, that I would make it section 7 and move everything else down.  I think making after the "10. Running Tests" doesn't give it the importance that it deserves.